src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexWriter.cs

```diff
@@ -399,21 +409,13 @@ private void EmitFragment(int nodetype, RegexNode node, int curIndex)
                     break;
 
                 case RegexNode.Require | BeforeChild:
-                    // NOTE: the following line causes lookahead/lookbehind to be
-                    // NON-BACKTRACKING. It can be commented out with (*)
```

Comment:
What did the old comment `It can be commented out with (*)` mean? Is this no longer true? Or is it too obvious for a comment?

Reply:
I have no idea :)  If it's referring to changing this line of code, obviously that's a breaking change in semantics.  And if it's referring to changing how the expression is written by the developer (which I presume it is), the comment isn't relevant here.